app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class
+                 * (self.clean_power
+                    - car.clean_mark)
+                 * self.average_rating
+                 / self.distance_from_city_center)
+        return price if price > 0 else 0
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            income += self.calculate_washing_price(car)
+
+            if self.wash_single_car(car):
```

Comment:
wash_single_car method suggests it should return None , but you are using an if statement to check the result of self.wash_single_car(car) as if it was returning a boolean or some other value that can be evaluated in a conditional.

The method should either:
Return a boolean value (for example, True if the car was washed successfully, and False otherwise), in which case the type hint and the method implementation need to be changed.
Or not be used in a conditional check if it's only performing an action without determining success/failure and thus truly returns None.

Reply:
regarding the code:
in the for loop there is an additional check to change the value of clean_mark for each machine.
![image](https://github.com/mate-academy/py-car-wash-station/assets/139710659/ffcb5aab-4d0d-4b5c-935d-9c070ef12ca8)

Reply:
not fixed. You should check `if self.clean_power > car.clean_mark` and then call method `wash_single_car`